components/boards/espressif_saola_1_wrover/board.h

```diff
@@ -45,7 +45,7 @@
 // GPIO connected to Neopixel data
 // Note: On the production version Saola (v1.2) it is GPIO 18,
 // however on earlier revision v1.1 it is GPIO 17
-#define PIN_NEOPIXEL          18
+#define PIN_NEOPIXEL          17
```

Comment:
production run saola board has neopixel on pin 18, you should keep this as local modification.